wordpress-mobile/gutenberg-mobile
# Add Array.prototype.flatMap polyfill

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3034

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/28400

## Description

Adds a polyfill for `Array.prototype.flatMap` function (built-in `flatMap`) because it's not fully supported on Android.

So far this wasn't needed because there were no references to that function but in a recent [PR](https://github.com/WordPress/gutenberg/pull/28371) in `gutenberg` it was introduced. It was added in the logic related to converting HTML to blocks, this functionality is used among other places in the Paste handler logic that is also used in the `RichText` component.

## To test
1. Add a paragraph block
2. Write some words and set some of them as bold or italic
3. Select all the text and copy it
4. Paste it in the same paragraph or another one
5. Check that no error is shown

PR submission checklist:
